README.md

```diff
@@ -29,6 +29,10 @@ Example Playbook
     sriov_devices:
       - name: p4p1
         numvfs: 63
+      - name: p3p1
+        numvfs: 8
+        # Don't add a udev rule to set numvfs
+        on_boot_configuration_enabled: false
```

Comment:
We need this setting for VF-LAG environments and if you're respinning this it may be helpful to document the purpose of this flag.